src/api/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:lts-alpine
+
+# Install pnpm
+RUN npm i -g pnpm
+
+# https://github.com/krallin/tini
+RUN apk --no-cache add tini
```

Comment:
This should be:

```dockerfile
# Tini Entrypoint for Alpine
# util-linux required to optimize builds using multiple cores
RUN apk add --no-cache tini util-linux
ENTRYPOINT [ "/sbin/tini", "--"]
```

By setting an `ENTRYPOINT` you are [defining the default command that gets run in the container](https://docs.docker.com/engine/reference/builder/#entrypoint).  In this case, we're saying, "use tini and have it receive its arguments from what comes next (--)", see https://unix.stackexchange.com/questions/11376/what-does-double-dash-mean.  The `tini` command will take care of initializing our node process for us.

A Dockerfile that uses this as a base would then define a `CMD` to [specify default params to our ENTRYPOINT](https://docs.docker.com/engine/reference/builder/#cmd):

```dockerfile
CMD ["node", "index.js"]
```

So in essence, we're really doing:

```sh
/sbin/tini node index.js
```

But we split it into stages, and the second stage can redefine what the actually arguments are via `CMD`.